packages/@coorpacademy-app-player/locales/en/player.json

```diff
@@ -1,38 +1,41 @@
 {
   "Access the lesson": "Access the lesson",
   "Back to home": "Back to home",
+  "Back to question": "Back to question",
+  "Bonus! Get an extra life by watching the lesson below!": "Bonus! Get an extra life by watching the lesson below!",
   "Check out the next chapter in this course!": "Check out the next chapter in this course!",
   "Clue": "Clue",
   "Coach": "Coach",
-  "Context": "Context",
   "Congratulations!": "Congratulations!",
+  "Context": "Context",
   "Correct answer": "Correct answer:",
   "Did you know that?": "Did you know that?",
+  "Game over": "Game over",
+  "Go to question": "Go to question",
   "Good answer": "Good answer",
   "Good job": "Good job",
   "Key point": "Key point",
   "Media": "Lesson",
-  "Next chapter_": "Next chapter:",
   "Next chapter": "Next chapter",
+  "Next chapter_": "Next chapter:",
   "Next level": "Next level",
   "Next level_": "Next level:",
   "Next": "Next",
   "Ooops": "Ooops...",
   "Ouch": "Ouch...",
   "Related subjects": "Related subjects:",
   "Retry chapter": "Retry chapter",
-  "Retry level": "Retry level",
   "Retry chapter_": "Retry chapter:",
+  "Retry level": "Retry level",
   "Retry level_": "Retry level:",
-  "Select something below": "Select something below",
+  "See your comment and those of your peers.": "See your comment and those of your peers.",
   "Select an answer": "Select an answer",
+  "Select something below": "Select something below",
+  "Share your opinion on this course": "Share your opinion on this course",
+  "Sorry, you have used your bonus!": "Sorry, you have used your bonus!",
+  "Thank you for your review !": "Thank you for your review !",
```

Comment:
`Thank you for your review!`

Et si on arrêtait avec les clés de traductions qui sont des phrases en anglais aussi ? :thinking:

Reply:
c'est un choix qu'on a fait au départ, on peut prevoir de tout changer d'un coup oui.

Reply:
Non non, les `__` c'est pour les tests seulement. Mais ça n'empêche que là y a une faute d'anglais à corriger

Reply:
tu parles bien du ` !` ?